examples/tutorials/curation/plot_2_train_a_model.py

```diff
@@ -0,0 +1,121 @@
+"""
+Training a model for automated curation
+=============================
+
+If the pretrained models do not give satisfactory performance on your data, it is easy to train your own classifier using SpikeInterface.
+"""
+import warnings
+warnings.filterwarnings("ignore")
+from pathlib import Path
+import spikeinterface.full as si
```

Comment:
I would import by module here, since it wil clearly show where the different functions are coming from